Source/UI/CommonUI/View/Component/InputView.swift

```diff
@@ -122,6 +121,10 @@ class InputView: UIView {
         errorView.update(style: style.error)
         errorView.isHidden = style.error?.isHidden ?? false
     }
+
+    func updateBorderColor(with color: UIColor) {
+        textFieldContainerBorder.updateBorderColor(to: color)
+    }
 }
```

Comment:
I think code is easier to navigate if we organise our logic by consistent criteria. In this case I would expect access control to play a part in the grouping of methods, and this internal method to appear before any private logic.